bin/node/testing/src/bench.rs

```diff
@@ -462,10 +463,10 @@ impl BenchKeyring {
 	}
 
 	/// Sign transaction with keypair from this keyring.
-	pub fn sign(&self, xt: CheckedExtrinsic, version: u32, genesis_hash: [u8; 32]) -> UncheckedExtrinsic {
+	pub fn sign(&self, xt: CheckedExtrinsic, spec_version: u32, tx_version: u32, genesis_hash: [u8; 32]) -> UncheckedExtrinsic {
```

Comment:
Why is this function not directly used from `node-testing`? It is already exposed and should not be duplicated for no reason.

Reply:
`node-testing` uses different keyring